### ISSUE TITLE
Chatterino freezes when you turn on mod mode for the first time.

### ISSUE THREAD

**Issue by somso2e** [Chatterino/chatterino2#2581]
**Describe the bug**
Chatterino freezes for less than a second when you turn on mod mode for the first time.

**To reproduce**
Turn off mod mode in every tab opened
Restart chatterino. 
You should see the the freezing when you turn it on again.

**Chatterino version**
Chatterino Nightly 2.2.3-beta2 (commit e00938dfd, )

**Operating system**
Windows 10

**Comment by jammehcow** [comment]
Unable to reproduce.

Chatterino 2.2.3-beta2 (commit `f9246a9e3`)
Arch Linux, Linux Zen 5.11.11-zen1

**Comment by apa420** [comment]
Unable to reproduce.
Chatterino Nightly 2.3.3 (commit 4dee7ad7, )
Windows 8.1

Does this still happen?
Otherwise I will close this.

**Comment by somso2e** [comment]
Yes, it still happens.... at least to me. I asked a few people that run windows too and none of them could reproduce it! I have no idea why it only happens to me. 

But at least I was able to find a more reliable way to reproduce it. As you can see in the screenshot, Chatterino only freezes and becomes unresponsive exactly when you try to load a normal user's message that has the moderation buttons next to their username. (not a mod or a broadcaster) 
![79A145EC-7801-4570-916C-224F6A93116E](https://user-images.githubusercontent.com/61581909/129921881-0682b8b6-d705-47d1-900c-81d75eb7ce59.GIF)


Chatterino Nightly 2.3.4 (commit f74b2770, )
Windows 10 Pro (20H2)

**Comment by ALazyMeme** [comment]
Cannot reproduce
Chatterino 2.3.4 (commit 4e5170799)
Windows 10 Pro (21H1)

**Comment by pajlada** [comment]
Still unable to reproduce, even with the latest steps.
If you're able to figure out what it is that's causing it, let us know so we can try to reproduce it.
I would've guessed you had a custom image or something for one of your mod actions that was slow to load, but that didn't seem to be it.
